File: output/CodeGen/codenet/C++/Java/s860472409.java
public class s860472409 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int[] a = new int[n];
        for (int i = 0; i < n; i++) {
            a[i] = in.nextInt();
        }
        in.close();
        int ans = 0;
        int count = 0;
        for (int i = 0; i < n - 1; i++) {
            if (a[i] == a[i + 1]) {
                count++;
                if (count % 2!= 0) {
                    ans++;
                }
            } else {
                count = 0;
            }
        }
        System.out.println(ans);
    }
}

File: output/CodeGen/codenet/C++/Python/s860472409.py
#!/usr/bin/python

import sys

def main():
    N = int(sys.argv[1])
    A = []
    for i in range(0,N):
        A.append(int(sys.argv[2]))
    i = 0
    ans = 0
    count = 0
    while A[i+1]!= None:
        if A[i] == A[i+1]:
            count += 1
            if count % 2!= 0:
                ans += 1
        else:
            count = 0
        i += 1
    print ans

if __name__ == '__main__':
    main()